provision/golang.sh

```diff
@@ -7,6 +7,7 @@ set -e
 sudo -u vagrant -E bash -c "mkdir ${GOPATH} && \
 go get -u github.com/google/gops && \
 go get github.com/subfuzion/envtpl/... && \
-go get -u github.com/gordonklaus/ineffassign"
+go get -u github.com/gordonklaus/ineffassign && \
+go get -u gitlab.com/opennota/check/cmd/..."
```

Comment:
seems related to #211 

With golangci-lint we can enable varcheck linter as per configuration file, it's quite flexible with linter settings as well. So what do you think ?

Reply:
You are right. golangci-lint has it already. I will close this PR.